message_definitions/v1.0/common.xml

```diff
@@ -4584,6 +4657,61 @@
       <field type="char[254]" name="text">Status text message, without null termination character.</field>
     </message>
+    <!-- Open Drone ID specific messages. The primary intention for these is to feed data to e.g. https://github.com/opendroneid/opendroneid-core-c -->
+    <message id="400" name="OPEN_DRONE_ID_BASIC_ID">
+      <wip/>
+      <!-- This message is work-in-progress it can therefore change, and should NOT be used in stable production environments -->
+      <description>Data for filling the OpenDroneID Basic ID message. See also https://github.com/opendroneid/opendroneid-core-c</description>
+      <field type="uint8_t" name="id_type" enum="OPENDRONEID_ID_TYPE">Indicates the format type for the ID of the UAS.</field>
+      <field type="uint8_t" name="uas_type" enum="OPENDRONEID_UAS_TYPE">Indicates the type of Unmanned Air/Aircraft System.</field>
+      <field type="uint8_t[21]" name="uas_type">UAS ID within the format of ID Type. Max length is 20 characters. If ASCII encoding is used, must be NULL terminated.</field>
+    </message>
+    <message id="401" name="OPEN_DRONE_ID_LOCATION">
+      <wip/>
+      <!-- This message is work-in-progress it can therefore change, and should NOT be used in stable production environments -->
+      <description>Data for filling the OpenDroneID Location message. The float data types below are 32-bit IEEE 754. The Location message provides the location, altitude, direction and speed of the drone.</description>
+      <field type="uint8_t" name="status" enum="OPENDRONEID_STATUS_TYPE">Indicates whether the UAS is on the ground or in the air.</field>
+      <field type="uint16_t" name="direction" units="cdeg">Direction over ground (NOT heading, but direction of movement) in degrees * 100, 0.0..359.99 degrees. If unknown, set to: UINT16_MAX</field>
+      <field type="uint16_t" name="speed_horizontal" units="cm/s">Ground speed. If unknown, set to: UINT16_MAX</field>
+      <field type="float" name="speed_vertical" units="m/s">The vertical speed. Up is positive.</field>
+      <field type="int32_t" name="latitude" units="degE7">Current latitude of the UAS.</field>
+      <field type="int32_t" name="longitude" units="degE7">Current longitude of the UAS.</field>
+      <field type="float" name="altitude_barometric" units="m">The altitude calculated from the barometric pressue. Reference is against 29.92inHg or 1013.2mb.</field>
+      <field type="float" name="altitude_geodetic" units="m">The geodetic altitude as defined by WGS84.</field>
+      <field type="uint8_t" name="height_reference" enum="OPENDRONEID_HEIGHT_REFERENCE">Indicates the reference point for the height field.</field>
+      <field type="float" name="height" units="m">The current height of the UAS above the take-off location or the ground as indicated by height_reference.</field>
+      <field type="float" name="horizontal_accuracy" units="m">The accuracy of the horizontal position.</field>
```

Comment:
Sorry, I don't understand. What unknown value are you referring to?

Reply:
If you don't have a position, or an accuracy estimate how do you report this in the MAVLink message?

I also apologize because I had written the comments along time ago, but apparently never hit the submit button on GitHub, and found the comments just sitting here.